python-client/giskard/cli.py

```diff
@@ -189,5 +187,36 @@ def _find_and_stop(is_server, host, port):
     remove_existing_pidfile(pid_file_path)
 
 
+def logs_options(fn):
+    fn = click.option(
+        "--lines",
+        "-n",
+        type=INT,
+        default=10,
+        help="Output the last N lines of the log file, 10 lines are displayed by default",
+    )(fn)
+    fn = click.option(
+        "--follow",
+        "-f",
+        "is_follow",
+        is_flag=True,
+        default=False,
+        help="Output appended data as new logs are being generated",
+    )(fn)
+    return fn
+
+
+@worker.command("logs")
+@logs_options
```

Comment:
you can inline log options with `@click.option` like it's done for stop command for example. A separate *_options method makes sense when you want to share same options for multiple commands